.github/ISSUE_TEMPLATE/missing-event.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Missing Event
+about: Request Implementation of a missing event
+title: ''
+labels: missing event
+assignees: Pjiesco
+
+---
+
+**Event Name**
+Please give the name of the event.
+
+**Description**
+If possible a discription of the event,
+
+**Event Fields**
```

Comment:
Replace by **Documentation**

Reply:
Do you still want to keep the Description header/field or do you want it replaced by the documentation? (same goes for the request template)

Reply:
Ah yeah, maybe not, I don't think it is relevant

Reply:
ok then i will replace it

Reply:
Another question: was it intentional to have the documentation header a different header size than the event fields? if so which headers should be bigger?

Reply:
Yes it was